### ISSUE TITLE
[Bug]: xfce4 vncserver doesnt work after update

### ISSUE THREAD

**Issue by babaric-dev** [termux/termux-packages#12139]
### Problem description

vncserver only shows black screen **after updating now**

### What steps will reproduce the bug?

Just upgrade the packages to latest version with `pkg up`
Install xfce4 and vncserver
Then start the vnc server

### What is the expected behavior?

xfce and vnc should be working normally

### System information

termux-info:

```
(irrelevant)
```

**Comment by babaric-dev** [comment]
The bug seems to be ahout the update of `libxfixes` and `dbus`.

**Comment by xtkoba** [comment]
Seems like dbus seccomp issue, maybe? I'll try to handle this.

**Comment by babaric-dev** [comment]
The libxfixes package removed `libXfixes.so.3` too:
```
CANNOT LINK EXECUTABLE "xfce4-session": library "libXfixes.so.3" not found
```

And downgrading `libxfixes` to `5.0.3-23` solves the error.

Conclusion: there are at least 2 problems in the update.

**Comment by babaric-dev** [comment]
> Seems like dbus seccomp issue, maybe? I'll try to handle this.

I don't know.. But here is the error message:
```
EOF in dbus-launch reading address from bus daemon
```

**Comment by xtkoba** [comment]
Should be fixed by `libxfixes` version `6.0.0-1` and `dbus` version `1.15.0-1`. Thanks!

**Comment by babaric-dev** [comment]
Great! Thanks for helping me sort out the bug!
